### DISCUSSION TITLE
Solving DLLNotFoundException: cvextern

### DISCUSSION THREAD

**Discussion by smshields** [emgucv/emgucv#421] [Q&A]
I am currently working on a unity project and attempting to import the x64 files. I can get the references to show up in my editor but I cannot make calls (always get the error DllNotFoundException). I can see the cvextern dll in my project, however. Has anyone else had this problem?

**Comment by emgucv** [comment]
Our official Emgu CV for unity release (currently at version 4.5.1) do not have this issue. You can get it from unity asset store. If you are interested in an evaluation copy, please feel free to email us at support@emgu.com

**Reply by smshields** [reply]
Hey there, I sent an email but have not received a reply. Could you help me get in touch?

**Reply by emgucv** [reply]
You email was replied on Feb 11, 9:06 EST, sent from canming@emgu.com 

Could it be possible that it has been filtered to SPAM by gmail?

**Reply by smshields** [reply]
Found it, you were right! Thank you for your response.
